keyboards/anne_pro/anne_pro_lighting.c

```diff
@@ -64,6 +66,29 @@ void anne_pro_lighting_init(void) {
     palSetPadMode(GPIOB, 11, PAL_MODE_ALTERNATE(7));
 }
 
+/* Handle state changes of keyboard LEDs (numlock, capslock) */
+void led_set_kb(uint8_t usb_led) {
+    /* Handle CapsLock led */
+    if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+        capslock_active = true;
+        /* Make sure the lighting chip is awake */
+        anne_pro_lighting_on();
```

Comment:
Calling twice (or more while its already woke up) the awake command turns on the backlight,a possible solution its to track if mcu is already woke up.

Reply:
`anne_pro_lighting_on()` already checks if the lighting mcu is awake and skips the call if it is. However it looks like the problem is if the mcu is awoken and only the capslock is turned on and then it is put back to sleep, and awoken again, then it turns on the backlight.

Reply:
another bug, when lt is used instead of kc_caps, if backlight is on, led just does the opposite.